### PR TITLE
Updating to work with Django 1.10

### PR DESCRIPTION
**Pull request by abolger**
I needed to make minor changes to use this with the present 'blessed' version of Django.

### CODE REVIEW

File: mongonaut/urls.py [hunk 1]
```diff
@@ -1,37 +1,77 @@
...
+        ),
+        url(
+            regex=r'^(?P<app_label>[_\-\w\.]+)/(?P<document_name>[_\-\w\.]+)/(?P<id>[\w]+)/delete/$',
+            view=views.DocumentDeleteView.as_view(),
+            name="document_delete"
+        )
     )
-)
+
+except Exception as e:
```
Comment (garrypolley) [inline mongonaut/urls.py]: This should cover a specific exception, not all of them.

File: mongonaut/utils.py [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
 
-from mongoengine.base import ObjectIdField, ValidationError
+from mongoengine.base import ObjectIdField
+try:
+    from mongoengine.base import ValidationError
+except:
```
Comment (garrypolley) [inline mongonaut/utils.py]: This should use `ImportError` like the one above.

**Comment by abolger** [comment]
New commit has fixed issues from code review.

File: mongonaut/urls.py [hunk 3]
```diff
@@ -1,37 +1,77 @@
...
-        name="document_list"
-    ),
-    url(
-        regex=r'^(?P<app_label>[_\-\w\.]+)/(?P<document_name>[_\-\w\.]+)/add/$',
-        view=views.DocumentAddFormView.as_view(),
-        name="document_detail_add_form"
-    ),
-    url(
-        regex=r'^(?P<app_label>[_\-\w\.]+)/(?P<document_name>[_\-\w\.]+)/(?P<id>[\w]+)/$',
+urlpatterns = []
```
Comment (garrypolley) [inline mongonaut/urls.py]: To clean this up do you mind putting all the `url` creations in this list and then exploding them out in the call inside the `try/catch`? 

For example:

```python
urlpatterns = [
    url() # All the patterns in this list.
]

try:
    from django.conf.urls import patterns

    urlpatterns = patterns('', *urlpatterns)
Except ImportError: 
    pass
```

**Comment by WoLpH** [comment]
I'm working on a new release which manually merges in several of these changes. Luckily most of the compatibility code isn't required anymore since the newer Django LTS releases have some compatibility code built in.

Thanks for the help!

**Pull request closed without merging**